portfolio/src/main/webapp/script.js

```diff
@@ -70,3 +71,10 @@ function createCommentElement(comment) {
     commentElement.innerText = comment.content;
```

Comment:
This function is of the `function foo() {}` form but the other functions are of the `const foo = function() {}` form.